geniza/common/management/commands/visual_tests.py

```diff
@@ -35,6 +35,9 @@ def take_snapshots(self, browser):
         browser.get("http://localhost:8000/documents/?q=physician")
         percy_snapshot(browser, "Document Search")
 
+        browser.get("http://localhost:8000/documents/?per_page=2")
+        percy_snapshot(browser, "Document Search (paginated)")
+
```

Comment:
Is there a way we can combine these two snapshots, or do we need both?

Reply:
Is my solution OK? Not the most meaningful search 😆 but at least it has a query and shows pagination with 5+ pages.

Reply:
😆 Thanks for working on consolidating this. It's ok to have a meaningless search for testing purposes! But can we also include a search term that gets highlighted in the description to show keywords-in-context styles? (I guess we didn't have that in the previous example...?)  Sorry this is kind of a pain. Maybe we can just add another search term.

Reply:
That was my intention, and oddly enough it _is_ highlighted on my local! I'll try adding a second keyword.

Reply:
Strange, I added a second keyword and it's still not highlighting the result in Percy. But it is working as expected for me locally!